{"name": "测试用例描述：错误的密码登录失败", "status": "passed", "description": "测试描述：错误的密码登录失败", "steps": [{"name": "输入错误的密码", "status": "passed", "attachments": [{"name": "错误的密码", "source": "3c03293d-d672-4844-b926-67f069bfff3c-attachment.attach"}], "start": 1651988422889, "stop": 1651988422889}], "attachments": [{"name": "截图", "source": "2a4421f8-b6cb-4001-bcc4-bc284817f3a8-attachment.png", "type": "image/png"}, {"name": "截图", "source": "64166f3f-b95d-42fc-b2e8-5965692fb118-attachment.png", "type": "image/png"}], "start": 1651988422889, "stop": 1651988424349, "uuid": "591ba83e-ada7-4624-8fd1-bd3ef74a72f2", "historyId": "e042998cd34bf7d92d802f115102d44e", "testCaseId": "ce17e49339328b1be4c6805d1bcb9a3f", "fullName": "test_two.Test#test_22", "labels": [{"name": "feature", "value": "登录模块"}, {"name": "epic", "value": "xxx测试"}, {"name": "story", "value": "登录"}, {"name": "severity", "value": "critical"}, {"name": "suite", "value": "test_two"}, {"name": "subSuite", "value": "Test"}, {"name": "host", "value": "MS-SLSXLICKSUSF"}, {"name": "thread", "value": "14400-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_two"}]}